setup/updating.md

```diff
@@ -38,11 +38,19 @@ Updating Coder is a two-step process:
     helm repo update
     ```
 
-2. Upgrade to the desired version (i.e., `1.16.0`):
+2. Export the current helm chart values into a file:
+
+    ```bash
+    helm get values --namespace coder coder > current-values.yml
+    ```
+
+3. Upgrade to the desired version with your helm chart values file (i.e., `1.16.0`):
+
+    _Note: Omitting `--version` will default to the latest version_
```

Comment:
should clarify whether it's the latest release version or latest release candidate, I think the former but it seems worth double checking

Reply:
@jawnsy who manages our helm repo? @deansheather ? Not sure how to know if this defaults to the latest release candidate or not, but I assume so?

Reply:
Merging as is now with approvals, but would love to hear some input on this for a future change to the doc.